models/iaf_psc_alpha.cpp

```diff
@@ -240,16 +236,85 @@ iaf_psc_alpha::calibrate()
 
   // these depend on the above. Please do not change the order.
   V_.P30_ = -P_.Tau_ / P_.C_ * numerics::expm1( -h / P_.Tau_ );
-
   V_.P21_ex_ = h * V_.P11_ex_;
-  V_.P31_ex_ = 1 / P_.C_ * ( ( V_.P11_ex_ - V_.P33_ ) / ( -1 / P_.tau_ex_ - -1 / P_.Tau_ )
-                             - h * V_.P11_ex_ ) / ( -1 / P_.Tau_ - -1 / P_.tau_ex_ );
-  V_.P32_ex_ = 1 / P_.C_ * ( V_.P33_ - V_.P11_ex_ ) / ( -1 / P_.Tau_ - -1 / P_.tau_ex_ );
-
   V_.P21_in_ = h * V_.P11_in_;
-  V_.P31_in_ = 1 / P_.C_ * ( ( V_.P11_in_ - V_.P33_ ) / ( -1 / P_.tau_in_ - -1 / P_.Tau_ )
-                             - h * V_.P11_in_ ) / ( -1 / P_.Tau_ - -1 / P_.tau_in_ );
-  V_.P32_in_ = 1 / P_.C_ * ( V_.P33_ - V_.P11_in_ ) / ( -1 / P_.Tau_ - -1 / P_.tau_in_ );
+
+  // exact propagator, linear approximation and singular case for excitatory synaptic event
+
+  const double P32_ex_linear = 1 / 2. / P_.C_ / P_.Tau_ / P_.Tau_ * h * h * ( P_.tau_ex_ - P_.Tau_ )
```

Comment:
I find this long sequence of divisions difficult to read, I think `1. / ( 2 * P_.C_ * P_.Tau_ * P_.Tau_ ) * h * h ...` would be clearer.